### PR TITLE
Flair app list now displays flair requirements on mouseover

### PR DESCRIPTION
**Pull request by not-an-aardvark**
[Example image](https://i.gyazo.com/e83f784defc0116b37c619ac116794a7.png)

This will probably conflict with your babeljs PR, do with it as you will.

### CODE REVIEW

File: assets/js/adminCtrl.js [hunk 1]
```diff
@@ -83,8 +83,54 @@ define([
             });
         };
 
+        $scope.getFlairs = function () {
```
Comment (YaManicKill) [inline assets/js/adminCtrl.js]: Question: don't we already do this somewhere?

File: assets/js/adminCtrl.js [hunk 2]
```diff
@@ -83,8 +83,54 @@ define([
...
+                  shinyevents: "",
+                  eggs: "",
+                  sub: ""
+                };
+              }
+            }
+          });
+        };
+
+        $scope.formattedRequirements = function (flair) {
```
Comment (YaManicKill) [inline assets/js/adminCtrl.js]: I like this, but why only for mods?

Comment (YaManicKill) [inline assets/js/adminCtrl.js]: Ah wait, I get it, it's for just the flair app list. Alright. Maybe we could do this for the app application as well?

File: assets/js/adminCtrl.js (hunk 1 above)
Comment (not-an-aardvark) [inline assets/js/adminCtrl.js]: We do, but it's in userCtrl.js. I wasn't sure if there was an easy way to share the functions between the clientside controllers.

File: assets/js/adminCtrl.js (hunk 2 above)
Comment (not-an-aardvark) [inline assets/js/adminCtrl.js]: Done, see the other commit.

**Pull request merged**